docs/docker-images/all-in-one.md

```diff
@@ -0,0 +1,76 @@
+This image contains `earthly`, `buildkit`, and some extra configuration to enable the two to work together. All that's missing is your source code! This image is mainly intended for use in containerized CI scenarios, or where maintaining a persistent installation of `earthly` isn't possible.
+
+## Quickstart
+
+Want to just get started? Here are a couple sample `docker run` commands that cover the most common use-cases:
+
+### Simple Usage
+
+```bash
+$ docker run --privileged -t -e NO_DOCKER=1 -v $(pwd):/workspace earthly/earthly:latest +for-linux
+```
+
+Heres a quick breakdown:
+
+- `--privileged` is required when you are using the internal `buildkit`. This is because `buildkit` currently requires is when used with `earthly`.
```

Comment:
```suggestion
- `--privileged` is required when you are using the internal `buildkit`. This is because `buildkit` currently requires it for overlayfs support.
```